src/iop/ashift.c

```diff
@@ -115,11 +115,11 @@ const char *name()
 
 const char *description()
 {
-  return _("distort perspective automatically,\n"
-           "for corrective and creative purposes.\n"
-           "works in RGB,\n"
-           "takes preferably a linear RGB input,\n"
-           "outputs linear RGB.");
+  return _("distort perspective automatically.\n\n"
```

Comment:
I would add `\t` after each icon to ensure text are aligned, it looks much better.

Reply:
Here is how it looks for filmicrgb:

![image](https://user-images.githubusercontent.com/467069/98905836-6745a080-24bc-11eb-9ec9-4b2a9844b200.png)

Reply:
it's cleaner but there is a lot of space between icon and text. Isn't there half tab or something like it ?

Reply:
maybe as an utf-8 character and could probably be done in the help function as proposed by AlicVB.